rero_ils/modules/cli.py

```diff
@@ -861,3 +869,64 @@ def reserve_pid_range(pid_type, records_number, unused):
                                              status=PIDStatus.NEW)
                 db.session.add(identifier(recid=pid))
             db.session.commit()
+
+
+@utils.command('runindex')
```

Comment:
How about just use `bulkindex` instead of `runindex`

Reply:
Because this is the command `run` the index -> invenio indexer `invenio index run`